code/obj/item/device/nuke_summon_remote.dm

```diff
@@ -0,0 +1,26 @@
+/obj/item/remote/nuke_summon_remote
+	name = "Nuclear Bomb Teleporter Remote"
+	icon = 'icons/obj/items/device.dmi'
+	desc = "A single-use teleporter remote that summons the nuclear bomb to the user's current location."
+	icon_state = "bomb_remote"
+	item_state = "electronic"
+	density = 0
+	anchored = 0.0
+	w_class = 2.0
+
+	var/charges = 1
+	var/use_sound = 'sound/machines/chime.ogg'
+
+	attack_self(mob/user as mob)
+		if(charges >= 1)
+			var/datum/game_mode/nuclear/mode = ticker.mode
+			showswirl(mode.the_bomb)
+			mode.the_bomb?.set_loc(get_turf(src))
```

Comment:
Might be nice to do this in a gamemode-agnostic way, to enable admin shenanigans.

Reply:
Maybe store the bomb as a var itself, that way the remote can be var-editied to any object?

Reply:
I just copy-pasted Pali's code for this because I don't have the brain power to do it myself.

Reply:
I made a pull request to implement my suggestion: https://github.com/Gannets/goonstation/pull/1

Reply:
![Xw65y1HRAg](https://user-images.githubusercontent.com/9563541/92687085-bd953800-f2ef-11ea-8545-1ab1d13954cf.gif) ![lLGJ81Lufj](https://user-images.githubusercontent.com/9563541/92687088-bff79200-f2ef-11ea-8f77-bcf3ffed3b7f.gif)